packages/Material-Core.package/MDSystemWindow.class/instance/initializeLabelArea.st

```diff
@@ -0,0 +1,29 @@
+as yet unclassified
+initializeLabelArea
+	"Initialize the label area (titlebar) for the window."
+	
+	labelString ifNil: [labelString := 'Untitled Window'].
+	label := MDLabel new contents: labelString.
+	label color: (MDColorTheme currentTheme textColorOn: MDColorTheme currentTheme primaryColor).
+			"Add collapse box so #labelHeight will work"
+			collapseBox := self createCollapseBox.
+			stripes := Array
+						with: (RectangleMorph newBounds: bounds)
+						with: (RectangleMorph newBounds: bounds).
+			"see extent:"
+			self addLabelArea.
+			self setLabelWidgetAllowance.
+			self addCloseBox.
+			self class moveMenuButtonRight 
+				ifTrue: [self addLabel. self addMenuControl]
+				ifFalse: [self addMenuControl. self addLabel].
+			self addExpandBox.
+			labelArea addMorphBack: collapseBox.
+			self setFramesForLabelArea.
+			Preferences noviceMode
+				ifTrue: [closeBox
+						ifNotNil: [closeBox setBalloonText: 'close window'].
+					menuBox
+						ifNotNil: [menuBox setBalloonText: 'window menu'].
+					collapseBox
+						ifNotNil: [collapseBox setBalloonText: 'collapse/expand window']].
```

Comment:
das ist jetzt ausschließlich copy-paste oder? Soll nicht am mergen hindern, aber da wollten wir ja eigentlich Funktionalität in die Titlebar verschieben